main.go

```diff
@@ -155,6 +155,13 @@ func loadConfigs() (config.Group, error) {
 		}
 	}
 
+	// Command line Arg
+	if flag.NArg() > 0 {
+		confString := strings.Join(flag.Args(), "\n")
+		confBody := ":" + config.Port + "\n" + strings.Replace(confString, ",", "\n", -1)
```

Comment:
I'll clarify what I mean. Spaces are significant in the Caddyfile. i.e. this:

```
:2015
browse gzip
```

is not valid (gzip needs to be on its own line) - it should be this:

```
:2015
browse
gzip
```

So newlines are required, but more cumbersome to type into the command line than commas are. Compare:

```
$ caddy "browse\ngzip"
```

with

```
$ caddy browse,gzip
```

And I think the second one is easier. True, it's not technically the same as the Caddyfile syntax you'd find in a file, but you can pipe those in if needed.